CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fixed an issue wich could lead to wrong Error messages due to a
+   race condition.
```

Comment:
actually this line is right, the others are not, since this will be backported i want to keep the formatting of the other lines as is